App/navigators/RootStack.tsx

```diff
@@ -37,6 +59,72 @@ const RootStack: React.FC = () => {
     navigation.navigate(Screens.Terms)
   }
 
+  const initAgent = async () => {
+    if (initAgentInProcess) {
+      return
+    }
+
+    Toast.show({
```

Comment:
As a convention we're not really doing any developer focused logging to the console. Plz remove in the future. I get Bifold is a reference wallet but most ppl won't want to show end users debug logs in the UI.